radix-engine-tests/assets/blueprints/metadata2/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+use scrypto::prelude::*;
+
+#[blueprint]
+mod metadata2 {
+    struct M {}
+
+    impl M {
+        pub fn f() {
+            let package_address = METADATA_MODULE_PACKAGE;
+            let blueprint_name = METADATA_BLUEPRINT;
+            let function_name = METADATA_CREATE_WITH_DATA_IDENT;
+
+            /*
```

Comment:
Is this commented-out code intended to be deleted? :)

Reply:
Ahh no it's explaining how the below was generated, my bad! All good.

Perhaps we could have a comment here briefly explaining that? But only if you feel like it, no worries, I was just being slow.

Reply:
Will some add notes.